Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml

```
@@ -54,6 +54,8 @@
   parent: BaseToolSurgery
   description: A surgical tool used to make incisions into flesh.
   components:
+  - type: Sharp
+  butcherDelayModifier: 1.5 # Butchering with a scalpel, regardless of the type, will take 50% longer
```

This won't get read properly unless the indentation is correct, `butcherDelayModifier` is a property of `SharpComponent` so it needs to be at the same indentation level.
```suggestion
  - type: Sharp
    butcherDelayModifier: 1.5 # Butchering with a scalpel, regardless of the type, will take 50% longer
```

---

Thanks for the assistance, I wasn't aware indents were part of the syntax!